sched/paging/pg_miss.c

```diff
@@ -179,7 +179,7 @@ void pg_miss(void)
   if (!g_pftcb)
     {
       pginfo("Signaling worker. PID: %d\n", g_pgworker);
-      nxsig_kill(g_pgworker, SIGWORK);
+      nxsig_kill(g_pgworker, SIGIO);
```

Comment:
This should be realtime signal.
Let's add `#define SIGWORK SIGRTMIN` into `sched/paging/paging.h` and use it

Reply:
The `SIGIO` has `nxsig_abnormal_termination` signal action

Reply:
Or maybe we can add call sequence similar to `sighold` for kernel task. Something like
```
sigemptyset(&set);
sigaddset(&set, SIGIO);
sigprocmask(SIG_SETMASK, &set, NULL);
```
For page worker

Reply:
Or change `nxthread_setup_scheduler()` and modify line `tcb->sigprocmask = rtcb->sigprocmask;` to become
```
if (ttype != TCB_FLAG_TTYPE_KERNEL)
  {
    tcb->sigprocmask = rtcb->sigprocmask;
  }
else
  {
    tcb->sigprocmask = ALL_SIGNAL_SET;
  }
```
Like, mask all signals for kernel threads. Does this makes sense?

Reply:
I mean that I do not expect that user can kill kernel thread at all.

Reply:
Ok, change to SIGPAGING(SIGRTMIN).